config.json

```diff
@@ -1135,6 +1135,19 @@
         "searching",
         "trees"
       ]
+    },
+    {
+      "uuid": "8740af44-002c-4716-a759-a68ae4c68737",
+      "slug": "isbn-verifier",
+      "core": false,
+      "unlocked_by": "bob",
```

Comment:
I am not sure about the order of exercises since nextercism, can other reviewers please check this?

Reply:
@matthewmorgan Can you have a look at this?

Reply:
This looks reasonable to me.  I don't think it needs to match the other tracks necessarily.  👍